### DISCUSSION TITLE
Getting the below error when running epinow forecast function. Please suggest what I can do.

### DISCUSSION THREAD

**Discussion by GauriSaran** [epiforecasts/EpiNow2#298] [General]
Chain 4: Rejecting initial value:
Chain 4:   Error evaluating the log probability at the initial value.
Chain 4: Exception: validate transformed params: R[26838] is 20.5012, but must be less than or equal to 20  (in 'model_estimate_infections' at line 59)

Chain 4: 
Chain 4: Initialization between (-2, 2) failed after 100 attempts. 
Chain 4:  Try specifying initial values, reducing ranges of constrained values, or reparameterizing the model.
[1] "Error in sampler$call_sampler(args_list[[i]]) : Initialization failed."
error occurred during calling the sampler; sampling not done
WARN [2022-04-04 18:41:06] epinow: some chains had errors; consider specifying chains = 1 to debug - .local, object, ...
here are whatever error messages were returned
[[1]]
Stan model 'estimate_infections' does not contain samples.

[[2]]
Stan model 'estimate_infections' does not contain samples.

[[3]]
Stan model 'estimate_infections' does not contain samples.

[[4]]
Stan model 'estimate_infections' does not contain samples.

Error: $ operator is invalid for atomic vectors
Called from: value[[3L]](cond)
Error during wrapup: unimplemented type (29) in 'eval'

Error: no more error handlers available (recursive errors?); invoking 'abort' restart
Error during wrapup: INTEGER() can only be applied to a 'integer', not a 'unknown type #29'
Error: no more error handlers available (recursive errors?); invoking 'abort' restart

**Comment by seabbs** [comment]
Hi @GauriSaran can you please post a full reprex that reproduces this behaviour?

**Reply by GauriSaran** [reply]
Hi this was resolved earlier. Sorry I should have closed this issue.

**Reply by seabbs** [reply]
No problem
